src/qibo/quantum_info/metrics.py

```diff
@@ -385,3 +385,92 @@ def gate_error(channel, target=None, backend=None):
     error = 1 - average_gate_fidelity(channel, target, backend=backend)
 
     return error
+
+
+def meyer_wallach(circuit, backend=None):
+    """Computes the Meyer-Wallach entanglement Q of the `circuit`,
+
+    .. math::
+        Q = 1-\\frac{1}{N}\\sum_{k}\\text{Tr}\\left(\\rho_k^2(\\theta_i)\\right) \\,
+
+    Args:
+        circuit (qibo.models.Circuit): Parametrized circuit.
+        backend (qibo.backends.abstract.Backend): Calculation engine.
+
+    Returns:
+        (int) : Meyer-Wallach entanglement.
```

Comment:
following the rest of the module, return type doesn't need to be inside parenthesis (all functions)